packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -461,8 +461,6 @@ class InspectorService extends InspectorServiceBase {
       'getPubRootDirectories',
     );
 
-    if (response.runtimeType != List) return [];
```

Comment:
You could use
```dart
if (response is! List<dynamic>) return []
```
You should also file a bug about the difference in VM behavior and Dart2JS behavior. I think the VM behavior is right and the type Object in Dart2Js should also be List.

Reply:
You are right, looks like that works!